qa/config/crimson_qa_overrides.yaml

```diff
@@ -1,7 +1,5 @@
 overrides:
   ceph:
-    log-ignorelist:
-    - \(POOL_APP_NOT_ENABLED\)
```

Comment:
Are you sure that this was added strictly to mop up the fallout and not because crimson, being a RADOS suite in nature, has grown a legitimate need?

Reply:
Given it was merged 5 days ago, I think it's fallout mop up.